R/company_technology_asset_value_at_risk.R

```diff
@@ -8,10 +8,13 @@
 #'   strongly the future dividends propagate to the company value
 #' @param flat_multiplier Numeric. A ratio that determines for the asset type
 #'   if how strongly the DCF should propagate to value changes.
+#' @param crispy Boolean. Indicates if the output should be used for the CRISPY
+#'   database or for standard portfolio calculation (default).
 company_technology_asset_value_at_risk <- function(data,
                                                    shock_scenario = NULL,
                                                    div_netprofit_prop_coef = NULL,
-                                                   flat_multiplier = NULL) {
+                                                   flat_multiplier = NULL,
+                                                   crispy = FALSE) {
```

Comment:
same as with lrisk, can we simply always calculate this and save us the if statement?

Reply:
or actually let us leave it as is, this way it will be easier to switch of the other calculations once they are no longer needed.